dask_cuda/cli/dask_cuda_worker.py

```diff
@@ -175,38 +181,36 @@
     help="Enable InfiniBand communication",
 )
 @click.option(
-    "--enable-rdmacm/--disable-rdmacm",
+    "--enable-nvlink/--disable-nvlink",
     default=False,
-    help="Enable RDMA connection manager, currently requires InfiniBand enabled.",
+    show_default=True,
+    help="Enable NVLink communication",
 )
 @click.option(
-    "--enable-nvlink/--disable-nvlink",
+    "--enable-rdmacm/--disable-rdmacm",
     default=False,
-    help="Enable NVLink communication",
+    show_default=True,
+    help="Enable RDMA connection manager, currently requires InfiniBand enabled.",
 )
 @click.option(
     "--net-devices",
     type=str,
     default=None,
-    help="When None (default), 'UCX_NET_DEVICES' will be left to its default. "
-    "Otherwise, it must be a non-empty string with the interface name, such as "
-    "such as 'eth0' or 'auto' to allow for automatically choosing the closest "
-    "interface based on the system's topology. Normally used only with "
-    "--enable-infiniband to specify the interface to be used by the worker, "
-    "such as 'mlx5_0:1' or 'ib0'. "
-    "WARNING: 'auto' requires UCX-Py to be installed and compiled with hwloc "
-    "support. Additionally that will always use the closest interface, and "
-    "that may cause unexpected errors if that interface is not properly "
-    "configured or is disconnected, for that reason it's limited to "
-    "InfiniBand only and will still cause unpredictable errors if not _ALL_ "
-    "interfaces are connected and properly configured.",
+    help="""If specified, workers will use this interface for UCX communication; can be
+    a string (like ``eth0``), or ``auto`` to pick the optimal interface based on
+    the system's topology; typically only used with ``--enable-infiniband``
+
+    .. warning::
+        ``auto`` requires UCX-Py to be installed and compiled with hwloc support;
+        unexpected errors can occur when using ``auto`` if any interfaces are
+        disconnected or improperly configured""",
```

Comment:
This is removing a lot of important information, such as the IB interface names that can be used, as well as "auto" being limited to IB only. Can we add that back?